### PR TITLE
Fix ENVVAR string interpolation

### PR DESCRIPTION
**Pull request by benjamintanweihao**

### CODE REVIEW

File: docs/source/installation/yatai_deployment.rst [hunk 1]
```diff
@@ -342,7 +342,7 @@ Store your ingress class in environment var:
 
 .. code:: bash
 
-  kubectl -n yatai-deployment patch cm/network --type merge --patch '{"data":{"ingress-class":"$INGRESS_CLASS"}}'
+  kubectl -n yatai-deployment patch cm/network --type merge --patch '{"data":{"ingress-class":"'${INGRESS_CLASS}'"}}'
```
Comment (yetone) [inline docs/source/installation/yatai_deployment.rst:345]:
```suggestion
  kubectl -n yatai-deployment patch cm/network --type merge --patch "{\"data\":{\"ingress-class\":\"${INGRESS_CLASS}\"}}"
```

File: docs/source/installation/yatai_deployment.rst [hunk 2]
```diff
@@ -443,7 +443,7 @@ You need to configure your DNS in one of the following two options:
         .. code:: bash
 
           # Replace yatai.example.com with your domain suffix
-          kubectl -n yatai-deployment patch cm/network --type merge --patch '{"data":{"domain-suffix":"yatai.example.com"}}'
+          kubectl -n yatai-deployment patch cm/network --type merge --patch '{"data":{"domain-suffix":"'${DOMAIN_PREFIX}'"}}'
```
Comment (yetone) [inline docs/source/installation/yatai_deployment.rst:446]: `$DOMAIN_PREFIX` This environment variable does not appear

Comment (benjamintanweihao) [reply docs/source/installation/yatai_deployment.rst:446]: This is already present in the existing document:

```
export DOMAIN_SUFFIX=yatai.example.com
```

**Comment by yetone** [comment]
LGTM

**Pull request merged**